.github/workflows/lint-test.yaml

```diff
@@ -173,10 +176,12 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/rust-setup
       - run: docker run --detach -p 5432:5432 cimg/postgres:14.2
+      - uses: taiki-e/install-action@v1.5.6
```

Comment:
what's this?

Reply:
this step just installs `cargo nextest` as a binary (instead of building it from scratch every time).